### PR TITLE
Added encryption configuration support

### PR DESCRIPTION
**Pull request by gnleong**
Added encryption configuration support of KMS encryption type with either assigned or a new KMS key

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,7 +134,9 @@ EOF
 | scan\_on\_push | Indicates whether images are scanned after being pushed to the repository (true) or not scanned (false). | `bool` | `true` | no |
 | tags | A mapping of tags to assign to the resource. | `map(string)` | `{}` | no |
 | timeouts | Timeouts map. | `map` | `{}` | no |
-| timeouts\_delete | Indicates whether images are scanned after being pushed to the repository (true) or not scanned (false). | `string` | n/a | no |
+| timeouts\_delete | How long to wait for a repository to be deleted. | `string` | n/a | no |
```
Comment (lgallard) [inline README.md:137]: Nice catch!

File: README.md [hunk 2]
```diff
@@ -134,7 +134,9 @@ EOF
...
+| encryption\_type | The encryption type to use for the repository. Valid values are `AES256` or `KMS` | `string` | `AES256` | no |
```
Comment (lgallard) [inline README.md:138]: I use terraform-docs to generate the provider, variables and outputs documentation, and it takes the values from the description in the variable definition. The terraform-docs tools sorts the variables as well.

Therefore, can you use the same description as in the variable definitions, and sort it alphabetically?  If it much trouble I can do it after merging your PR.

**Review by lgallard — COMMENTED** [review]
Thanks for you PR. I just added a couple of comments before merging it!

File: README.md (hunk 2 above)
Comment (gnleong) [reply README.md]: Updated a sorted input with corresponding variable descriptions.

**Comment by lgallard** [comment]
@gnleong thanks for the PR, I already merged it!

**Pull request merged**